src/Learning/KWLearningProblem/KWLearningProblemComputeStats.cpp

```diff
@@ -116,9 +116,7 @@ void KWLearningProblem::ComputeStats()
 	KWLearningErrorManager::AddTask("Data preparation");
 
 	// Initialisation des objets de calculs des statistiques
-	bPreparationOk = bPreparationOk and not TaskProgression::IsInterruptionRequested();
```

Comment:
Why can we afford not to keep the test
```c++
if (not TaskProgression::IsInterruptionRequested())
{
    InitializeClassStats(classStats, &learningSpec);
}
```

This means that even if interruption is requested, we nonetheless move on to initializing the statistic computation objects?

Reply:
C'est vrai en théorie, mais cela complexifie inutilement le code
Et il n'y pas de gros enjeu: ici, on initialise essentiellement une spécification d'apprentissage, qui sera disponible quoi il arrive tout au long du reste de la méthode.
Cf. commentaire précédent

Reply:
OK, as the updated comment now reflects this choice.